examples/elm/index.html

```diff
@@ -1,34 +1,31 @@
-<!doctype html>
-<html lang="en" data-framework="elm">
+<!DOCTYPE HTML>
+<html>
+
 <head>
-	<meta charset="utf-8">
-	<title>Elm • TodoMVC</title>
-	<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
-	<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
-	<style>
-		body {
-			width: auto;
-		}
-		.todomvc-wrapper {
-			visibility: visible !important;
-		}
-	</style>
+  <meta charset="UTF-8">
```

Comment:
Could you retain the tabs? Some more rules are outlined here: https://github.com/tastejs/todomvc/blob/master/codestyle.md